app/views/AboutSimilarGoodsView.scala.html

```diff
@@ -75,7 +75,7 @@
         @hmrcCharacterCount(CharCount(
             id = "value",
             name = "value",
-            maxLength = Some(8000),
```

Comment:
why have you changed the maxLength? For one the form provider will kick up a stink if the lengths are different but the ticket this was introduced in specified 8000 characters